bom/pom.xml

```diff
@@ -38,11 +38,13 @@
     <influxdb.version>1.2.4-alpine</influxdb.version>
     <jackson.version>2.9.0</jackson.version>
     <java-base-image.name>openjdk:8u162-jre-slim</java-base-image.name>
-    <jjwt.version>0.7.0</jjwt.version>
+    <jaxb.api.version>2.3.0</jaxb.api.version>
+    <javax.annotation.api.version>1.3.2</javax.annotation.api.version>
+    <jjwt.version>0.9.0</jjwt.version>
```

Comment:
we cannot simply update this to 0.9.0 without a CQ

Reply:
I know.